tests/plotting/bokeh/testgridplot.py

```diff
@@ -7,7 +7,8 @@
 from .testplot import TestBokehPlot, bokeh_renderer
 
 try:
-    from bokeh.models import Div
+    from bokeh.layouts import Column, Row
+    from bokeh.models import Div, ToolbarBox
```

Comment:
``Row` is imported here but I don't see it used? Won't that be a flake?

Reply:
Yes, good catch.